ipv8/messaging/anonymization/community.py

```diff
@@ -763,11 +763,17 @@ def on_created(self, source_address, payload, _):
             self.directions[request.from_circuit_id] = EXIT_NODE
             self.remove_exit_socket(request.from_circuit_id)
 
-            self.send_cell([relay.peer], u"extended", ExtendedPayload(relay.circuit_id,
-                                                                      payload.key,
-                                                                      payload.auth,
-                                                                      payload.candidate_list_enc))
-        elif self.request_cache.has(u"retry", payload.circuit_id):
+            self.send_cell(
```

Comment:
I'm not a fan of this formatting style, please change it back.

Reply:
I did it like this because the indentation was a flake8 violation for some reason, but will do.

Reply:
Reverted